### DISCUSSION TITLE
I Hate My Nokia

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#16] [Comments Archive]
# I Hate My Nokia

My review of my Nokia G22. A phone touted as being easily repairable that doesn't quite live up to the standards it sets for itself.

https://vale.rocks/posts/i-hate-my-nokia

**Comment by DeclanChidlow** [comment]
As I went to post this article I [saw this post on the fediverse](https://social.tchncs.de/@maxxx/111809579815880076) which indicates the phones are bricking themselves en masse. How wonderful...

**Comment by DeclanChidlow** [comment]
I got my last promised *monthly* security update at the beginning of January. We're now approaching mid April. That's 3 and a half months.
